### PR TITLE
Added Album Art to Collection View

### PR DESCRIPTION
**Pull request by 0xcaff**
I've added a responsive album art loader which grabs album art from iTunes. Here are some screenshots:

screenshot 2015-08-13 at 7 14 07 pm
screenshot 2015-08-13 at 7 14 55 pm

### CODE REVIEW

File: app/components/album-art.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/album-art';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ["album-art"],
+  classNameBinding: ["loading"],
```
Comment (morgoth) [inline app/components/album-art.js]: I'm getting errors and lot of warnings on Ember 2.0. Will investigate it later.

File: app/components/album-art.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  getSize: function() {
+    this.set("size", $("img", this.element).width());
+  }.on("didInsertElement"),
+
+  getImage: function() {
+    var album = this.get("album");
+    try {
```
Comment (morgoth) [inline app/components/album-art.js]: why try-catch here? conditional that checks if album have image would be enough

File: app/components/album-art.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      this.set("imageSource", album.images[0]);
+    } catch (_) {
+      $.getJSON(`http://itunes.apple.com/search?term=${encodeURIComponent(album.name)}&limit=1&entity=album&callback=?`)
```
Comment (morgoth) [inline app/components/album-art.js:17]: Do you have some example so I can test fetching it? I tried some queries, but the album picture url was always returned altogether with album object

Comment (0xcaff) [inline app/components/album-art.js:17]: [Here is the reference for this API.](http://stackoverflow.com/questions/11250434/what-free-api-has-the-largest-collection-of-albums-covers#answer-11250511)

File: app/components/album-art.js (hunk 2 above)
Comment (0xcaff) [inline app/components/album-art.js]: Fixed.

File: app/components/album-art.js (hunk 1 above)
Comment (0xcaff) [inline app/components/album-art.js]: I re-based on top of master and couldn't reproduce. Maybe it was some thing to do with the try-catch block?

Comment (0xcaff) [inline app/components/album-art.js]: I'm getting strange errors too. I pulled the latest version but didn't `npm`/`bower` `install`.

Comment (0xcaff) [inline app/components/album-art.js]: Fixed

**Comment by morgoth** [comment]
Thanks. I removed size fetching as it doesn't make any visible difference to me.

**Comment by 0xcaff** [comment]
It should improve the quality on hires displays but whatever.

**Pull request merged**